packages/protocol/contracts/verifiers/RiscZeroVerifier.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "../common/EssentialContract.sol";
+import "../L1/ITaikoL1.sol";
+import "./IVerifier.sol";
+import "./libs/LibPublicInputHash.sol";
+
+/// @notice Verifier interface for RISC Zero receipts of execution.
+/// https://github.com/risc0/risc0-ethereum/blob/release-0.7/contracts/src/IRiscZeroVerifier.sol
```

Comment:
I suggest we move this IRiscZeroVerifier defination to `thirdparty/risc0/IRiscZeroVerifier.sol` and our RiskZeroVerifeir.sol shall implement it, not taking it as a parameter for `init`.

Reply:
Sure i can move it, but what shall it implement.. ? It is an external contract, we need to call it, there is no workaround about calling the RisckZero verifier contract.

Reply:
Where is the actual verifier code? It's our code or thirdparty code?